bundles/org.eclipse.jface/src/org/eclipse/jface/preference/ScaleFieldEditor.java

```diff
@@ -317,6 +317,10 @@ private void updateScale() {
 		if (scale != null && !scale.isDisposed()) {
 			scale.setMinimum(getMinimum());
 			scale.setMaximum(getMaximum());
+			// Reapplying the minimum to ensure that the scale's value is correctly
```

Comment:
It’s better to a this comment because otherwise is looks like a redundant mistake.

Reply:
I don't understand what you mean.

Reply:
I think it is just a confirmation that this comment is important. :)

Reply:
Yes, sorry for the typo.  Everything is so small on an iphone....